sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -276,63 +243,53 @@ async fn receive_events(client: &ConsumerClient) -> Result<(), Box<dyn std::erro
             }
         }
     }
+
     Ok(())
 }
 ```
 
 # Troubleshooting
 
+## General
+
+When you interact with the Azure Event Hubs client library using the Rust SDK, errors returned by the service correspond to the same HTTP status codes returned for [REST API] requests.
```

Comment:
This isn't strictly accurate - we return `azure_core::Error` values using `ErrorKind::Other` which are `azure_messaging_eventhubs::Error` values.

The `get_eventhubs_properties()` and `get_eventhubs_partition_properties()` APIs return HTTP status codes but they are buried and hard to find (in the current design). This will be fixed in a future iteration of the eventhubs and amqp clients.

Reply:
I wrote something new based on what you said. Please double check me cause I'm not too sure what I wrote is accurate. 
Also, we don't need to be super extensive here. We'll have time to improve on the troubleshooting sections as the libraries change and improve.